.dockerignore

```diff
@@ -0,0 +1,8 @@
+kube
```

Comment:
It exposes your architecture and it isn't of interest to the wider open-source community.
I'm sure that @timgent will have a better idea about it than me.